src/fill_pg_plugin.cpp

```diff
@@ -711,33 +711,37 @@ struct fpg_session : connection_callbacks, std::enable_shared_from_this<fpg_sess
         unsigned numRows = 0;
         varuint32_from_bin(num, bin);
         for (uint32_t i = 0; i < num; ++i) {
-            check_variant(bin, get_type("table_delta"), "table_delta_v0");
-            table_delta_v0 table_delta;
-            from_bin(table_delta, bin);
+            uint32_t index;
+            eosio::varuint32_from_bin(index, bin);
+            table_delta t_delta;
+            from_bin(t_delta, bin);
```

Comment:
This double-reads the variant's index, causing corruption. `index` isn't needed.

Reply:
The varuint32_from_bin is replacing a read in this location formerly done by `check_variant`.  Was that an unexpected read?

Reply:
The previous code:
* `check_variant` (read the index)
* read a `table_delta_v0` (not a variant - doesn't read the index)

Reply:
Understood.  Removed.